Dockerfile

```diff
@@ -2,11 +2,13 @@ FROM fedora:30
 RUN dnf install -y \
     npm \
     chromium \
-    && npm install -g lighthouse \
     && mkdir -p /root/.config/configstore \
     && echo '{"isErrorReportingEnabled": false}' > /root/.config/configstore/lighthouse.json
 
 WORKDIR /opt/lighthouse
+
+RUN npm install lighthouse
+
```

Comment:
Does it work to do this in the same place it used to, just without the `-g`? Because I think that reduces the number of Docker layers (or at least used to).

Reply:
https://github.com/iFixit/lighthouse-docker/pull/6/commits/88372225477fa126d2a82c6fef848e516002a088